PxWeb/appsettings.Release.json

```diff
@@ -40,8 +40,9 @@
     },
     "AdminProtection": {
       "IpWhitelist": [ "#{ADMIN_IPCONFIG}#" ],
-      "AdminKey": "#{API_ADMIN_KEY}#"
-    },
+      "AdminKey": [ "#{API_ADMIN_KEY}#" ]
```

Comment:
AdminKey is a string so there should not be any square brackets

Reply:
have now adjusted this

Reply:
looks good now.